src/main/java/net/fabricmc/loader/impl/util/SystemProperties.java

```diff
@@ -33,7 +37,4 @@
 	public static final String DEBUG_DISABLE_MOD_SHUFFLE = "fabric.debug.disableModShuffle";
 	// workaround for bad load order dependencies
 	public static final String DEBUG_LOAD_LATE = "fabric.debug.loadLate";
-
-	private SystemProperties() {
```

Comment:
Uh, why remove this

Reply:
It is pointless outside api classes

Reply:
Uh, from a point of saving code, you should have just made this an interface so we can omit all those `public static final`s...

Reply:
it is not about saving code, but not adding redundant bits, an interface would convey a wrong purpose.

Reply:
Sure, this doesn't affect code behavior after all